prisma/migrations/20240426144110_email_hash/migration.sql

```diff
@@ -0,0 +1,43 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[emailHash]` on the table `users` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- AlterTable
+ALTER TABLE "users" ADD COLUMN     "emailHash" TEXT;
+
+-- CreateIndex
+CREATE UNIQUE INDEX "users.email_hash_unique" ON "users"("emailHash");
+
+-- hack ... prisma doesn't know about our other schemas (e.g. pgboss)
+-- and this is only really a problem on their "shadow database"
+-- so we catch the exception it throws and ignore it
+CREATE OR REPLACE FUNCTION submit_migrate_existing_user_emails_job() RETURNS void AS $$
+    BEGIN
+        -- Submit a job to salt and hash emails after the updated worker has spun-up
+        INSERT INTO pgboss.job (name, data, priority, startafter, expirein)
+        SELECT 'saltAndHashEmails', jsonb_build_object(), -100, now() + interval '10 minutes', interval '1 day';
```

Comment:
Copy/paste from a different job insertion.

Conceptually I understand the purpose of `priority` but in practice, I'm not sure of the implications, especially in this case. I suppose this job might be higher priority than something like re-indexing items, so I could see an argument to set it to something higher than `-100`, but am definitely open to input.

Reply:
> I'm not sure of the implications, especially in this case

Ah ok, me neither haha, I just expected to not see an explicit priority in that case since you don't have to set it yourself

Reply:
Yea, I copied it from the indexing bookmarked items migration job.

This is probably a good candidate to run at default priority.